### ISSUE TITLE
blender error 

### ISSUE THREAD

**Issue by ronnyascencio** [masqu3rad3/tik_manager4#103]
**Describe the bug**

**To Reproduce**
just opening blender

**Expected behavior**

**Screenshots**
![Screenshot from 2024-04-27 22-46-22](https://github.com/masqu3rad3/tik_manager4/assets/95187946/98e8dc4e-ba4f-4550-913b-70555b0ab465)

**DCC**
Please complete the following information:
- Dcc name(s): blender
- Dcc Version(s) 4.1

**Desktop (please complete the following information):**
 - OS: linux
 - Version rocky 9.3

**Additional context**
opening blender

**Comment by masqu3rad3** [comment]
Hi,

This is an installation issue. It basically cannot find the tik_manager because the python path looks like not initialized.?

Can you double check the installation following the steps described here:
https://tik-manager4.readthedocs.io/en/latest/installation/blender.html

**Comment by masqu3rad3** [comment]
Did you edit the:
```
tik_path = "PATH\\TO\\TIK_MANAGER4\\"
```
line in the tik_4_init.py ?? (line 11)
The issue seems there. It is either the wrong path or not edited at all.

**Comment by ronnyascencio** [comment]
i did it 
![Screenshot from 2024-04-28 16-54-45](https://github.com/masqu3rad3/tik_manager4/assets/95187946/b0633ecb-5f1d-499f-aefe-1433909de8ca)

**Comment by masqu3rad3** [comment]
The error in your previous message clearly says it cannot import the tik_manager4 module.
We cannot expect anything work after that error.

Please make sure the path is correct.

You should be able to 'import tik_manager4' in the python environment before expecting anything to work.

First resolve tht issue. If there are Pyside2 issues, that will come later.

**Comment by ronnyascencio** [comment]
Hi if you can see the I follow the documentation and now is not showing nothing no error no nothing I'll start a fresh re install and let you know

**Comment by ronnyascencio** [comment]
Hello so i have this now blender is able to know where the module is but still not showing the menu of tikmanager 
![Screenshot from 2024-04-28 20-17-34](https://github.com/masqu3rad3/tik_manager4/assets/95187946/cb173755-d934-4477-b960-5fcc305616c3)

**Comment by ronnyascencio** [comment]
Okay i manage to solve  get the tik manager menu now when i hit the tik UI or any other menu in tik manager menu i got this :
![Screenshot from 2024-04-28 20-49-19](https://github.com/masqu3rad3/tik_manager4/assets/95187946/1726908f-80f9-44fe-bd33-8bd3ea7345fc)

I have installed PySide2 do i need PyQt5?

**Comment by masqu3rad3** [comment]
You can try the PyQt5.
This looks like an issue with PySide2 version indeed.

Apologies, for Windows, I provide a frozen version of PySide2 which I know it works but for Linux this requires a bit debugging and relies a bit on technical problem solving.

I'll ask others who runs it on Linux. My main testing environment is Windows.

I reckon this may be one of another your super specific issues.
Btw, you probably already realized it but there is a typo in your command that you screenshot 2 messages ago:
```import tik_manaager4```
It should be:
```import tik_manager4```

**Comment by ronnyascencio** [comment]
well i did a fresh install of all and installed pyside 2 and PyQt5 too and got this :
![Screenshot from 2024-04-29 10-18-14](https://github.com/masqu3rad3/tik_manager4/assets/95187946/69306e1e-6396-42e3-b000-1909251db739)

**Comment by ronnyascencio** [comment]
this is the path i think is on the blender python interpreter
/home/rascencio/Apps/OpenSource/blender-4.1.1/4.1/python/lib/python3.11/site-packages/PyQt5
/home/rascencio/Apps/OpenSource/blender-4.1.1/4.1/python/lib/python3.11/site-packages/PySide2

**Comment by masqu3rad3** [comment]
Qt.py picks up pyside2 since it has prio and both available. Try to remove PySide2 and you can give another try.

**Comment by ronnyascencio** [comment]
I removed pyside2 and then open blander and run tik ui and bleder close 
![Screenshot from 2024-04-29 11-11-17](https://github.com/masqu3rad3/tik_manager4/assets/95187946/0bd54236-4710-4565-8709-9aebd1f17c93)

**Comment by masqu3rad3** [comment]
Something is wrong and it is beyond the scope of Tik Manager.
Can you bring back the PySide2 and this time copy/paste the error message instead of a screenshot? We can google the results.

You can put the pasted text betweet triple ` chars for code view:
```like this```

**Comment by ronnyascencio** [comment]
```
[rascencio@ronny ~]$ blender
Read prefs: "/home/rascencio/.config/blender/4.1/config/userpref.blend"
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
Traceback (most recent call last):
  File "/home/rascencio/Apps/OpenSource/blender-4.1.1/4.1/scripts/startup/tik_4_init.py", line 26, in execute
    tik_ui = main.launch(dcc="Blender")
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rascencio/Apps/Piepeline/tik_manager4/tik_manager4/ui/main.py", line 62, in launch
    m = MainUI(tik, parent=parent, window_name=window_name)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rascencio/Apps/Piepeline/tik_manager4/tik_manager4/ui/main.py", line 87, in __init__
    _style_file = pick.style_file()
                  ^^^^^^^^^^^^^^^^^
  File "/home/rascencio/Apps/Piepeline/tik_manager4/tik_manager4/ui/pick.py", line 30, in style_file
    style_qfile.open(QtCore.QFile.ReadOnly | QtCore.QFile.Text)
TypeError: 'PySide2.QtCore.QIODevice.OpenModeFlag' object cannot be interpreted as an integer
Error: Python: Traceback (most recent call last):
  File "/home/rascencio/Apps/OpenSource/blender-4.1.1/4.1/scripts/startup/tik_4_init.py", line 26, in execute
    tik_ui = main.launch(dcc="Blender")
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rascencio/Apps/Piepeline/tik_manager4/tik_manager4/ui/main.py", line 62, in launch
    m = MainUI(tik, parent=parent, window_name=window_name)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/rascencio/Apps/Piepeline/tik_manager4/tik_manager4/ui/main.py", line 87, in __init__
    _style_file = pick.style_file()
                  ^^^^^^^^^^^^^^^^^
  File "/home/rascencio/Apps/Piepeline/tik_manager4/tik_manager4/ui/pick.py", line 30, in style_file
    style_qfile.open(QtCore.QFile.ReadOnly | QtCore.QFile.Text)
TypeError: 'PySide2.QtCore.QIODevice.OpenModeFlag' object cannot be interpreted as an integer
```

**Comment by masqu3rad3** [comment]
which version of PySide2 did you deployed?

```
import PySide2
print(PySide2.__version__)
```

This should return the version.

The one I frozen for Windows is '5.15.2.1'

**Comment by ronnyascencio** [comment]
```[rascencio@ronny ~]$ /home/rascencio/Apps/OpenSource/blender-4.1.1/4.1/python/bin/python3.11 -m pip show PySide2
Name: PySide2
Version: 5.13.2
Summary: Python bindings for the Qt cross-platform application and UI framework
Home-page: https://www.pyside.org
Author: Qt for Python Team
Author-email: pyside@qt-project.org
License: LGPL
Location: /home/rascencio/Apps/OpenSource/blender-4.1.1/4.1/python/lib/python3.11/site-packages
Requires: shiboken2
```
confirming throgh blener consol
```
>>> import PySide2
>>> print(PySide2.__version__)
5.13.2

>>> 
```

**Comment by masqu3rad3** [comment]
will you be able to update to 5.15.2 to see if its version related?

**Comment by ronnyascencio** [comment]
Okay what i am thinking now is that is because of the version of blender  this is what i have when i tried to install the 5.15.2
```
[rascencio@ronny ~]$ /home/rascencio/Apps/OpenSource/blender-4.1.1/4.1/python/bin/python3.11 -m pip install PySide2==5.15.2
ERROR: Ignored the following versions that require a different python version: 5.14.0 Requires-Python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.9; 5.14.1 Requires-Python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.9; 5.14.2 Requires-Python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.9; 5.14.2.1 Requires-Python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.9; 5.14.2.2 Requires-Python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.9; 5.14.2.3 Requires-Python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.9; 5.15.0 Requires-Python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.9; 5.15.1 Requires-Python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.10; 5.15.2 Requires-Python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.10; 5.15.2.1 Requires-Python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.11
ERROR: Could not find a version that satisfies the requirement PySide2==5.15.2 (from versions: 0.0.0a1, 5.11.0, 5.11.1, 5.11.2, 5.12.0, 5.12.1, 5.12.2, 5.12.3, 5.12.4, 5.12.5, 5.12.6, 5.13.0, 5.13.1, 5.13.2)
ERROR: No matching distribution found for PySide2==5.15.2
[rascencio@ronny ~]$ /home/rascencio/Apps/OpenSource/blender-4.1.1/4.1/python/bin/python3.11 -m pip install PySide2
Requirement already satisfied: PySide2 in ./Apps/OpenSource/blender-4.1.1/4.1/python/lib/python3.11/site-packages (5.13.2)
Requirement already satisfied: shiboken2==5.13.2 in ./Apps/OpenSource/blender-4.1.1/4.1/python/lib/python3.11/site-packages (from PySide2) (5.13.2)
[rascencio@ronny ~]$ 

```

**Comment by masqu3rad3** [comment]
I am not sure. On windows I am testing it on Blender 4.1 too.

**Comment by ronnyascencio** [comment]
okay so i installed blender 4.0.2
and it runs with 5.15.2 

it loads the gui but on black 

![Screenshot from 2024-04-29 12-33-35](https://github.com/masqu3rad3/tik_manager4/assets/95187946/f0c78c6e-4f2c-4ae0-85a8-dc325a8ae89c)

**Comment by ronnyascencio** [comment]
no im deploying within 
i installed 4.1.0 and has the same problem of 4.1.1  and this versions do not allow 5.15 just the 4.0 alows it because it has python 3.10 and versions 4.1+ is python 3.11

**Comment by masqu3rad3** [comment]
I am curious how you install PySide2. Via which command etc.

**Comment by ronnyascencio** [comment]
i am installing it from the blender  python environment  with this command on command line
```
/home/rascencio/Apps/OpenSource/blender-4.1.0/4.1/python/bin/python3.11 -m pip install PySide2 
```
or if is the 4.0 just changing the comand to :
```
/home/rascencio/Apps/OpenSource/blender-4.0.2/4.0/python/bin/python3.10 -m pip install PySide2
``
3.10 install the 5.15.2
and 
3.11 install the 5.13.2

dont know what is going on hahahaha

**Comment by masqu3rad3** [comment]
Can you try installing PySide6 ?
first remove the PySide2

`/home/rascencio/Apps/OpenSource/blender-4.0.2/4.0/python/bin/python3.10 -m pip uninstall PySide2`

then
`/home/rascencio/Apps/OpenSource/blender-4.0.2/4.0/python/bin/python3.10 -m pip install PySide6`

**Comment by ronnyascencio** [comment]
Hello sorry gfor the dilay  i installed Pyside6 and ir didnt crash but have this 
![Screenshot from 2024-05-02 19-00-26](https://github.com/masqu3rad3/tik_manager4/assets/95187946/1a878d93-cde9-47e9-bf3a-34cdd52b2c01)

**Comment by masqu3rad3** [comment]
Yes I installed the Rocky Linux on a virtual machine, got exactly the same issue. No error, messages like its working but doesn't pop up ui.
I wanted to eliminate that it might be something VM related, thanks for testing that.

On another note, I tested launching the standalone UI on Rocky with PySide2 and PyQt5, none of them works. Only PySide6 works for the standalone UI.

I am not a Linux guru but this is something probably related with this distro.
